homeassistant/components/loqed/config_flow.py

```diff
@@ -41,27 +95,62 @@ async def async_step_zeroconf(
         # Check if already exists
         await self.async_set_unique_id(lock_data["bridge_mac_wifi"])
         self._abort_if_unique_id_configured({CONF_HOST: host})
-        return await self.async_step_user()
 
-    async def async_oauth_create_entry(self, data: dict) -> FlowResult:
-        """Create an entry for the flow.
+        return await self.async_step_user()
 
-        Ok to override if you want to fetch extra info or even add another step.
-        """
-        session = async_get_clientsession(self.hass)
-        res = await session.request(
-            "GET",
-            "https://app.loqed.com/API/integration_oauth3/retrieve_lock_data.php",
-            headers={"Authorization": f"Bearer {data['token']['access_token']}"},
+    async def async_step_user(
+        self, user_input: dict[str, Any] | None = None
+    ) -> FlowResult:
+        """Show userform to user."""
+        user_data_schema = vol.Schema(
+            {
+                vol.Required(CONF_API_TOKEN): str,
+            }
         )
-        lock_data = await res.json(content_type="text/html")
+        if user_input is None:
+            return self.async_show_form(
+                step_id="user",
+                data_schema=user_data_schema,
+                description_placeholders={
+                    "config_url": "https://integrations.production.loqed.com/personal-access-tokens",
```

Comment:
I don't fully understand why this is configurable. Isn't this url static for all Loqed users ? Or am I missing something ?

Reply:
It's a non-translatable string. I was told to not put those in the strings.json

Reply:
I was wondering why to show the URL in the first place. What benefit does it have to the user to show this (rather lengthy) url ?

Reply:
My Idea was to allow the user to click on it and create the API key if they don't have one yet. Just like you do in the google_sheets integration for example.

Reply:
ah, now I see it, that was not very clear to me. I was under the impression that by continuing the integration would create the token there. Not enough coffee today I guess.

Reply:
I updated the documentation PR as well to reflect this new flow.

Reply:
I've tested the flow and it works. It's still a bit wonky flow imo but that is out of your hands, functionality wise it works as it should.